### DISCUSSION TITLE
Reset form controller value in stacked form

### DISCUSSION THREAD

**Discussion by sanjai-mahalingan** [FilledStacks/stacked#1048] [Q&A]
Hi, i'm new to stacked, can you help me to about form value reset.

**Comment by zuboje** [comment]
@sanjai-mahalingan, assuming you are using `TextEditingController`, you can use `.clear()` method on your controller. For example if you created field like this:

```
@FormView(fields: <FormTextField>[
  FormTextField(
      name: 'myTextField',
])
```

you would call this method to reset value

`myTextFieldController.clear()`
